drgn_tools/rds.py

```diff
@@ -1020,6 +1117,421 @@ def rds_sock_info(
         return None
 
 
+def rds_print_recv_msg_queue(
+    prog: drgn.Program,
+    laddr: Optional[str] = None,
+    raddr: Optional[str] = None,
+    tos: Optional[str] = None,
+    lport: Optional[str] = None,
+    rport: Optional[str] = None,
+    ret: Optional[bool] = False,
+    outfile: Optional[str] = None,
+    report: bool = False,
+) -> None:
+    """
+    Print the rds recv msg queue similar rds-info -r
+
+    :param prog: drgn program
+    :param laddr: comma seperated string list of LOCAL-IP.  Ex: '192.168.X.X, 10.211.X.X, ...'
+    :param raddr: comma seperated string list of REMOTE-IP.  Ex: '192.168.X.X, 10.211.X.X, ...'
+    :param tos: comma seperated string list of TOS.  Ex: '0, 3, ...'
+    :param lport: comma seperated string list of lport.  Ex: 2259, 36554, ...'
+    :param rport: comma seperated string list of rport.  Ex: 2259, 36554, ...'
+    :param outfile: A file to write the output to.
+    :param report: Open the file in append mode. Used to generate a report of all the functions in the rds module.
+    :returns: None
+
+    """
+    msg = ensure_debuginfo(prog, ["rds"])
+    if msg:
+        print(msg)
+
+    index = -1
+    msg_queue: List[List[Any]] = [
+        [
+            " ",
+            "LocalAddr",
+            "LPort",
+            "RemoteAddr",
+            "RPort",
+            "Tos",
+            "Seq",
+            "Bytes",
+        ]
+    ]
```

Comment:
It would be cool to start using `Table` rather than `print_table()`. It's very similar but more flexible. And if necessary, it can be converted to have immediate output (at the expense of possible misaligned columns).

```suggestion
    print("Receive message queue:")
    msg_queue = Table(
        [
            " ",
            "LocalAddr",
            "LPort",
            "RemoteAddr",
            "RPort",
            "Tos",
            "Seq",
            "Bytes",
        ],
        outfile=outfile,
        report=report,
    )
```